.spelling

```diff
@@ -479,6 +479,54 @@ embeddable
 p99
 vCPU
 AES-NI
+3.99ms
+72.96ms
+18x
+8x
+appswitch
+appswitch-0
+AppSwitch
+bitpipe
+DaemonSet
+datapath
+decap
```

Comment:
??

Reply:
Remove this from the dictionary, not a word